docs/languages.md

```diff
@@ -53,11 +60,50 @@ the app can't find a translation of the string.
 
 ## Languages Other than English and Spanish
 
-Most of the navigation, buttons, and general wording throughout Orchid has been
-pulled out into strings in `config/locales/en.yml` and `config/locales/es.yml`.
-Translate each entry of the YAML file into your needed language(s).
-You may toggle between languages in the application and view the
-language differences as you go.
+Orchid supports English and Spanish internationalization out of the box.
+However, in many cases, you may need to add other languages.
+
+### Adding the Language Toggle
+
+When multiple languages are supported, Orchid displays a toggle which allows
+users to select their preferred website language.
+
+If adding a language which is not English or Spanish, you will need
+to add the text of this toggle. Copy `config/locales/languages.yml` to that
+same location in your app. You will need to add your language's name to the
+`&langs` definition, and make it available to your language's locales configuration.
+
+For example, if you were adding Japanese, your
+`config/locales/languages.yml` file would be:
+
+```yaml
+langs: &langs
+  en: English
+  es: Español
+  ja: 日本語
```

Comment:
I think it's more common to abbreviate Japan(ese) as `jp`. `ja` might be Java(nese)

Reply:
Oh interesting.  I was using the LoC list here, although the abbreviation can be arbitrary so it's not like we need to stick to a particular authority list or anything:  https://www.loc.gov/standards/iso639-2/php/code_list.php

Reply:
Oh okay. Works for me